karcioszki-ui/src/app/layout/dialogs/game-create-lobby-dialog.html

```diff
@@ -10,11 +10,11 @@ <h1 mat-dialog-title class="flex-center-vertical position-relative">
 <div mat-dialog-content class="flex column">
     <div>
         <mat-form-field>
-            <input matInput i18n-placeholder="@@lobbyGameId" placeholder="Game ID" [(ngModel)]="data.gameId" [ngModelOptions]="{standalone: true}">
+            <input matInput i18n-placeholder="@@lobbyGameId" placeholder="Game ID" [disabled]="'true'" [(ngModel)]="data.gameId" [ngModelOptions]="{standalone: true}">
         </mat-form-field>
         <mat-form-field>
             <mat-label i18n="@@lobbyCardCount">Card count:</mat-label>
-            <input matInput i18n-placeholder="@@lobbyCardCountInput" placeholder="Enter a card limit..." [(ngModel)]="data.cardCount"
+            <input matInput i18n-placeholder="@@lobbyCardCountInput" [type]="'number'" placeholder="Enter a card limit..." [(ngModel)]="data.cardCount"
```

Comment:
`[type]="'number'"` can be changed to `type="number"`. We do not have to provide a string variable, we can use just a plain attribute instead.  

See [angular tutorial](https://radek2s.github.io/Framework-Hero/angular/) in *Component Parameters > Passing properties* section. There is a simple input that receive a string.